### PR TITLE
SCAL-200657 add support for date ,number and currency formatting

### PR DESCRIPTION
**Pull request by abey-thomas**

### CODE REVIEW

File: src/embed/ts-embed.spec.ts [hunk 1]
```diff
@@ -1291,6 +1291,28 @@ describe('Unit test case for ts embed', () => {
...
         });
+
+        it('Should set the override locale for number/date and currency format', async () => {
+            jest.spyOn(baseInstance, 'getAuthPromise').mockResolvedValue(true);
+            init({
+                thoughtSpotHost: 'tshost',
+                authType: AuthType.None,
+                numberFormatLocale: 'en-US',
+                dateFormatLocale: 'en-IN',
+                currencyFormat: 'USD',
```
Comment (ashubham) [inline src/embed/ts-embed.spec.ts:1302]: is `currenyFormat` the right name? Should this be just `currency` ? As that what it looks to be set to

Comment (abey-thomas) [reply src/embed/ts-embed.spec.ts:1302]: this expects ISO codes @ashubham , based on it, we will use the libraries to show valid currency. Hence using Format as a key

File: src/types.ts [hunk 2]
```diff
@@ -592,6 +592,24 @@ export interface EmbedConfig {
...
+    /**
+     * Overrides default/user preffered locale for date formatting
+     *
+     * @version SDK: 1.28.0 | Thoughtspot: 10.0.0.cl, 9.5.0.sw
+     */
+    dateFormatLocale?: string;
+    /**
+     * Overrides default/user preffered locale for number formatting
+     *
+     * @version SDK: 1.28.0 | Thoughtspot: 10.0.0.cl, 9.5.0.sw
```
Comment (sastaachar) [inline src/types.ts:604]: update all sdk versions , 1.28.3 is already published so later than that

Comment (abey-thomas) [reply src/types.ts]: Done

**Review by sastaachar — APPROVED** [review]
lgtm!!

**Pull request merged**